addons/ui/org.openhab.ui.cometvisu/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package:
  javax.servlet,
  javax.servlet.annotation,
  javax.servlet.http,
+ javax.sql,
```

Comment:
you have only added javax.sql, but @maggu2810 mentioned in https://github.com/openhab/openhab2-addons/issues/533#issuecomment-163923637 also javax.cache. Are you sure that you do not need that as well?